app/services/appointment_notification_service.rb

```diff
@@ -11,12 +11,12 @@ def initialize(appointments:, days_overdue: 3, schedule_at:)
   end
 
   def send_after_missed_visit
-    overdue_with_patient_consent = appointments.overdue_by(days_overdue)
-                        .includes(patient: [:phone_numbers])
-                        .where(patients: { reminder_consent: 'granted' })
-                        .merge(PatientPhoneNumber.phone_type_mobile)
+    eligible_appointments = appointments
+                             .eligible_for_reminders(days_overdue: days_overdue)
```

Comment:
@prabhanshuguptagit Any reason why you didn't squeeze up 

```
.includes(patient: [:phone_numbers]).merge(PatientPhoneNumber.phone_type_mobile)
```

into the `eligible_for_reminders` definition?

I feel like freezing the complete definition as we currently understand it is valuable in one place. We can't send  reminders to folks who don't have mobile numbers anyway (as per our current understanding).

wdyt?

Reply:
> We can't send reminders to folks who don't have mobile numbers anyway (as per our current understanding).

That makes sense. The reason to leave it out is to let the call site decide what number it wants to use (or if it even wants to include the number) but it definitely belongs in there given this.

Reply:
Added in 9c512a4